### DISCUSSION TITLE
Some icons not show correctly

### DISCUSSION THREAD

**Discussion by khaitranhq** [kyazdani42/nvim-web-devicons#425] [Q&A] [answered]
![image](https://github.com/nvim-tree/nvim-web-devicons/assets/24852182/3ccb0b06-1103-4ab9-be75-7f93b08b0704)
When using nvim-tree, some icons display only a rectangle, like the one in the image for prettier. The git status icons of nvim-tree also show a rectangle.
I tried many Nerd fonts but nothing changed.
```
OS: Arch Linux
Neovim: v0.9.5
Terminal: Kitty
Shell: fish
nvim-web-devicons: { "branch": "master", "commit": "cb0c967c9723a76ccb1be0cc3a9a10e577d2f6ec" }
Current Font: FiraCode Nerd Font
```

**Comment by gegoune** [comment, marked as answer]
What are the other icons that don't display properly? I think Prettier icons were added recently and perhaps your font is not of latest upstream version.

**Comment by khaitranhq** [comment]
**Updated** I checked again and the git status icons of nvim-tree.lua is good now. Prettierrc is still a rectangle.
I tried downloading and installing FireCode Nerd again but it doesn't work. How can I check which font is updated your icons?

**Comment by alex-courtis** [comment]
I _think_ prettier is still awaiting release: https://github.com/nvim-tree/nvim-web-devicons/pull/417

**Reply by khaitranhq** [reply]
Thank you, I patched FiraCode with the new icons and it worked
